### DISCUSSION TITLE
Error when running MSNoise: Can't connect to MySQL server

### DISCUSSION THREAD

**Discussion by olivialocke** [ROBelgium/MSNoise#309] [Q&A] [answered]
Hi, 

I am attempting to install MSNoise using the MSNoise Documentation Release 1.6 (http://msnoise.org/doc/MSNoise.pdf). 

I get stuck on Step 6 of 1.1.1 Full Installation (msnoise bugreport) with this error: 

CRITICAL:root:Unable to read project configuration: error connecting to the database:
(pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on 'None' ([Errno 11001] getaddrinfo failed)")
(Background on this error at: http://sqlalche.me/e/14/e3q8)

I have checked that MySQL is running (version 8.0.34). I am working on Win 11 x64 and have tried several python versions. 
Unsure how to proceed - has anyone had a similar issue? 

Thanks!

**Comment by ThomasLecocq** [comment, marked as answer]
Hey,

If your project is relatively small, you should just be fine with SQLite. I'm rewriting the MySQL/PostgreSQL documentation for the new version

Thomas

**Reply by olivialocke** [reply]
Thank you for the quick reply! 

I am an undergrad and a bit out of my depth - 

I downloaded SQLite and started it but when I run msnoise bugreport I get the same error as before (still saying cannot connect to MySQL).

Is there something I need to do so that MSNoise recognizes SQLite? Or, is there somewhere I can find documentation on how to install MSNoise using SQLite?

**Reply by ThomasLecocq** [reply]
You don't have to install anything to use SQLite, just use 1. SQLite when running the msnoise dB init command

**Reply by olivialocke** [reply]
Ok - that makes sense! 

But, any commands that start with msnoise (msnoise bugreport, msnoise test, msnoise db ...) all give the same error: 

CRITICAL:root:Unable to read project configuration: error connecting to the database:
(pymysql.err.OperationalError) (2003, "Can't connect to MySQL server on 'None' ([Errno 11001] getaddrinfo failed)")
(Background on this error at: http://sqlalche.me/e/14/e3q8)

**Reply by ThomasLecocq** [reply]
You have to start from a clean folder (aka one without a previous dB.ini file)

**Reply by olivialocke** [reply]
That solved the error - thank you so much!!
